quisp/rules/Rule.cc

```diff
@@ -30,6 +30,8 @@ void Rule::setAction(Action *a) {
 }
 cPacket *Rule::checkrun(cModule *re) {
   cPacket *pk = nullptr;
+
+  // DEBUG
```

Comment:
what do you mean?

Reply:
This is just an old comment. I removed it.